app/controllers/backend/exports_controller.rb

```diff
@@ -18,11 +18,21 @@
 
 module Backend
   class ExportsController < Backend::BaseController
+    include Backend::ExportsHelper
```

Comment:
If the helper in in the standard helper directory, you can access its methods with `helpers.<method>' in the controllers, no need to include it.

Reply:
done!!